west.yml

```diff
@@ -108,7 +108,7 @@ manifest:
       repo-path: sdk-mcuboot
       revision: 34b3ac78780f861fb6a6eeffa42e6c7242c4eec9
       path: bootloader/mcuboot
-    - name: mbedtls-nrf
+    - name: mbedtls
```

Comment:
we need to get an update in nrfxlib first here:
https://github.com/nrfconnect/sdk-nrfxlib/blob/898ad8d8b57b520150a6308f386655ef1b326a40/common.cmake#L87

This stems back from the days when `mbedtls-nrf` was pointing directly at vanilla mbedtls from arm repo, and not Zephyr's fork.

Done this way because mbedtls from arm was not a Zephyr module, and the Zephyr module version of mbedtls was to old for nRF Security.

cc @frkv